src/index.html

```diff
@@ -10,13 +10,102 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo_img"
+      >
+        <img
+          src="images\logo.png"
```

Comment:
The image source path uses a backslash (`\`), which is incorrect for web URLs. It should be a forward slash (`/`) to ensure the image loads correctly across all browsers. Change `images\logo.png` to `images/logo.png`.